packages/google_sign_in/google_sign_in/example/ios/RunnerTests/GoogleSignInTests.m

```diff
@@ -16,7 +16,7 @@ @interface FLTGoogleSignInPluginTest : XCTestCase
 @property(strong, nonatomic) NSObject<FlutterBinaryMessenger> *mockBinaryMessenger;
 @property(strong, nonatomic) NSObject<FlutterPluginRegistrar> *mockPluginRegistrar;
 @property(strong, nonatomic) FLTGoogleSignInPlugin *plugin;
-@property(strong, nonatomic) GIDSignIn *mockSharedInstance;
+@property(strong, nonatomic) id mockSharedInstance;
```

Comment:
Can we give this variable a clearer name, especially since it no longer has a type?

Reply:
Done.